src/ControlSystem/Protocols/Submeasurement.hpp

```diff
@@ -34,9 +34,9 @@ namespace control_system::protocols {
 /// - `ControlSystems /*meta*/`
 ///
 /// where the last two are generally template parameters to the
-/// function.  Calling this function on every chare contributing to
-/// the measurement must result in a single call on one chare to
-/// `control_system::RunCallbacks<ConformingStructBeingDefinedHere,
+/// function.  This function will be called on every element, and
+/// these calls must collectively result in a single call on one chare
```

Comment:
Clarify that this "one chare" where the single call is can be any chare (not necessarily the elements or the control system chares). It just has to run *somewhere*. That confused me for a long time.